config.ini

```diff
@@ -90,6 +90,12 @@ log_level = DEBUG
 authentication = false
 db_connection = ${db_driver}://${db_user}:${db_pass}@${db_host}:${db_port}/${db_name}
 db_schema = ras
+debug = true
+rabbit_name = local_rabbit
+rabbit_host = localhost
+rabbit_port = 1234
```

Comment:
Can we default this to the out of the box deployment for rabbit when installed locally?, should make it easier for developers
rabbit_port = 15672
rabbit_username = guest
rabbit_password = guest